packages/bolt-connection/test/bolt/bolt-protocol-v5x1.test.js

```diff
@@ -0,0 +1,1092 @@
+/**
```

Comment:
Not for this PR but i wonder if we could have these work off each other somehow, on the server we have Annotations that say run this test on x,y,z versions to stop duplication

Reply:
I don't bother to have duplication in the test code, it make it easier to address differences and make the tests totally decoupled.

Maybe it worth to add subset of re-usable tests per capability, it can be a interesting re-factory to do.